apps/web/pages/_app.tsx

```diff
@@ -27,6 +28,18 @@ const calFont = localFont({
 function MyApp(props: AppProps) {
   const { Component, pageProps, err, router } = props;
   let pageStatus = "200";
+  const { cssVarsPerTheme } = useEmbedUiConfig();
+  const cssVarsStyle = [];
+  if (cssVarsPerTheme) {
+    for (const [themeName, cssVars] of Object.entries(cssVarsPerTheme)) {
+      cssVarsStyle.push(`.${themeName} {`);
+      for (const [cssVarName, value] of Object.entries(cssVars)) {
+        cssVarsStyle.push(`--${cssVarName}: ${value};`);
+      }
+      cssVarsStyle.push(`}`);
+    }
+  }
```

Comment:
Can we reuse useCalTheme here in anyway ?

packages/ui/styles/useCalcomTheme.tsx

Reply:
Nice will check that

Reply:
useCalComTheme is doing that in useEffect and needs document to work. I am currently generating style string which works on server as well. So, that won't be usable here.